### ISSUE TITLE
Class 'Guzzle\Http\Url' not found

### ISSUE THREAD

**Issue by romainmoyne** [jkphl/micrometa#20]
Hi

When trying that library, i got this message:

```
 [message] => Class 'Guzzle\Http\Url' not found
 [file] => vendor/jkphl/dom-factory/src/Domfactory/Infrastructure/Dom.php
```

The thing is I have upgraded Guzzle to 6.2.3 so it should work. It looks like Guzzle has not that class anymore.

Any idea?

**Comment by jkphl** [comment]
@romainmoyne Minor correnction: The "error" obviously doesn't occur in *micrometa* but rather in [jkphl/dom-factory](https://github.com/jkphl/dom-factory). Besides, it looks you're kinda right: The current code seems to use a class of the old and abandoned [guzzle/guzzle](https://github.com/guzzle/guzzle3) package. Not sure why this installed at all (all tests are running fine at my end though). I'll pursue this over at the dom-factory repo.

**Comment by jkphl** [comment]
@romainmoyne Should be fixed as of latest release (2.0.1). Can you confirm that?

**Comment by romainmoyne** [comment]
@jkphl I cannot test at the moment but I trust you :)
